stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -556,10 +564,10 @@ function doRequest(string url, http:Request request, http:ClientConfiguration cl
     }
 }
 
-# Prepare the request to be sent to the authorization endpoint by adding the relevant headers and payloads.
+# Prepares the request to be sent to the authorization endpoint by adding the relevant headers and payloads.
 #
-# + config - `RequestConfig` record
-# + return - Prepared HTTP request object or `Error` if an error occurred during preparing request
+# + config - `oauth2:RequestConfig` record
+# + return - Prepared HTTP request object or `oauth2:Error` if an error occurred during preparing request
```

Comment:
```suggestion
# + return - Prepared HTTP request object or `oauth2:Error` if an error occurred during preparing request
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - Prepared HTTP request object or else `oauth2:Error` if an error occurred while preparing the request
```